index.html

```diff
@@ -285,6 +285,55 @@ <h2>
       <!-- menu and about page end -->
     </div> 
 
+<!-- footer/form start-->
+  <section>
+  <footer class="contact-form">
+    <div class="contact-heading">
+      <h2 class="contact-h2">
+         I'm always<br>
+         intersted in<br>
+         hearing about<br>
+         new projects,so<br>
+         if you'd like to<br>
+         chat please get<br>
+         in touch.
+      </h2>
+    </div>
+    <form action="https://formspree.io/f/cnqrnjgr"  class="contact-details" method="post">
+        <ul class="contact-list">
+            <li>
+                <input type="text" class="contact-1" maxlength="30" placeholder="Full name" required>
+            </li>
+            <li>
+                <input type="email" class="contact-2" placeholder="Email address" required>
+            </li>
+            <li>
+                <textarea class="contact-3" maxlength="500" placeholder="Enter text here" required></textarea>
+            </li>
+        </ul>
+        <div class="contact-button">
+            <button type="submit" class="button-submit">
+                Get in Touch
+            </button>
+        </div>
```

Comment:
- [ ] As stated in the Project requirement I recommend you to Implement the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
  
![Capture](https://user-images.githubusercontent.com/93598189/193869162-6f890854-0663-4da6-a594-f0dc9cf5f993.PNG)

Reply:
- [ ] As said by the previous reviewer, Kindly consider implementing the buttons interaction. `hover`, `disable`, and `active` for it helps the user to see the action they are about to make on your page.